### PR TITLE
Add SUDT example

### PR DESCRIPTION
**Pull request by duanyytop**

### CODE REVIEW

File: example/src/main/java/org/nervos/ckb/SUDTExample.java [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+  }
+
+  private static String transfer() throws IOException {
+    List<ScriptGroupWithPrivateKeys> scriptGroupWithPrivateKeysList = new ArrayList<>();
+
+    TransactionBuilder txBuilder = new TransactionBuilder(api);
+    IndexerCollector txUtils = new IndexerCollector(api, ckbIndexerApi);
+
+    List<Receiver> receivers =
+        Collections.singletonList(new Receiver(ReceiveAddresses.get(0), Utils.ckbToShannon(150)));
```
Comment (shaojunda) [inline example/src/main/java/org/nervos/ckb/SUDTExample.java:141]: Would it be better to use 142 here?

File: example/src/main/java/org/nervos/ckb/SUDTExample.java [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+  }
+
+  private static String issue() throws IOException {
+    List<ScriptGroupWithPrivateKeys> scriptGroupWithPrivateKeysList = new ArrayList<>();
+
+    TransactionBuilder txBuilder = new TransactionBuilder(api);
+    IndexerCollector txUtils = new IndexerCollector(api, ckbIndexerApi);
+
+    List<Receiver> receivers =
+        Collections.singletonList(new Receiver(SendAddresses.get(0), Utils.ckbToShannon(400)));
```
Comment (shaojunda) [inline example/src/main/java/org/nervos/ckb/SUDTExample.java:84]: Why is 400 here? Is it just a random value? The minimum capacity of a sudt cell is 142 ckb, right?

File: example/src/main/java/org/nervos/ckb/SUDTExample.java (hunk 1 above)
Comment (duanyytop) [reply example/src/main/java/org/nervos/ckb/SUDTExample.java:141]: Why？I think this example is not used to illustrate the minimum capacity of sudt.

File: example/src/main/java/org/nervos/ckb/SUDTExample.java (hunk 2 above)
Comment (duanyytop) [reply example/src/main/java/org/nervos/ckb/SUDTExample.java:84]: I will display how to transfer sudt from the sender address using `transfer` method, so I set a bigger number as capacity.

File: example/src/main/java/org/nervos/ckb/SUDTExample.java (hunk 1 above)
Comment (shaojunda) [reply example/src/main/java/org/nervos/ckb/SUDTExample.java:141]: no more than 142 in real scenarios.

Comment (duanyytop) [reply example/src/main/java/org/nervos/ckb/SUDTExample.java]: Updated in https://github.com/nervosnetwork/ckb-sdk-java/pull/381/commits/14b9fe5e0b76be7f7983287f122c9d4f6bb58118

**Review by shaojunda — APPROVED** [review]

**Pull request merged**